asterius/package.yaml

```diff
@@ -100,49 +100,49 @@ executables:
   ahc:
     source-dirs: app
     main: ahc.hs
-    ghc-options: -threaded -rtsopts
+    ghc-options: -threaded -rtsopts "-with-rtsopts=-A64m -n2m -I0 -qg"
```

Comment:
Couldn't we use a top-level `ghc-options` used by all the executables instead of copy-pasting it?

Reply:
iirc that will emit GHC warnings for the library target.

Reply:
We could use anchors and aliases as mentioned in the docs: https://github.com/sol/hpack#not-repeating-yourself The example they give is exactly what we want to do.